src/paths/properties@average@group-by-{groupBy}.yaml

```
@@ -0,0 +1,29 @@
+get:
+  tags:
+    - Indicateurs sur les biens (ancien/neuf)
+  summary: Moyenne (GroupBy)
+  description: |-
+    Permet de récupérer des moyennes sur les biens regroupés selon 1 critère.
+
+    ### Exemple
+      #### Prix au m² moyen regroupés par portail des appartements en cours de publication en vente à Levallois-Perret
+        /properties/average/group-by-crawl-sources?field=PRICE_M2&marketingTypes=SALE&active=true&zipCodes=92300&source=AD&propertyTypes=APARTMENT
+  parameters:
+    - $ref: ../components/parameters/PropertyGroupByCountQuery-groupBy.yaml
+    - $ref: ../components/parameters/PropertyGroupByCountQuery-termsSize.yaml
+    - name: propertySearchQuery
+      in: query
+      description: Paramètres de recherche de biens
+      schema:
+        $ref: ../components/schemas/PropertySearchQuery.yaml
+  responses:
+    '200':
+      description: Moyenne
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/AverageGroupBy.yaml  # TODO fix response
```

:sparkles: suggestion: Utiliser `Average.yaml` et supprimer `AverageGroupBy.yaml`, ou bien fixer les réponses des endpoints group-by :
```suggestion
            $ref: ../components/schemas/Average.yaml  # TODO fix response
```